scale_linear:
  x: -0.005
  y: 0.005
  z: 0.005
  
scale_angular:
  x: -0.05
  y: 0.05
  z: 0.05
